perl6/lib/Cutie.pm

```diff
@@ -1,3 +1,3 @@
 role Cutie {
-  has Bool $.accept-hugs is rw = False;
+  method accept-hugs { … }
```

Comment:
Ellipsis characters are also valid constructor for lazy list objects, eg `my @laaazy = 1…Inf;`